boards/posix/native_posix/doc/board.rst

```diff
@@ -537,8 +537,16 @@ The following peripherals are currently provided with this board:
 .. _SDL2:
    https://www.libsdl.org/download-2.0.php
 
+**Flash driver**:
+  A flash driver is provided that access all flash data through a binary file
+  on the host file system.
```

Comment:
It feels like a lot more documentation could help the user.
What size is that file? is the file content just the raw binary dump? Are pages emulated? a nice drawing maybe?

Reply:
I agree with @aescolar.  More information about this file is needed.  Does it have to exist before starting, or will it be created (as an empty file)?  Can you read or write past the end of file, or is the file exactly the flash size specified somewhere and you'll get some memory fault if reading beyond the file/flash size limit?  What's the recommended way to create this flash.bin file?